egs/wsj/s5/steps/data/gen_wav_sample.py

```diff
@@ -0,0 +1,103 @@
+#encoding:utf8
+from __future__ import print_function
+import subprocess
+import argparse
+import re
+
+def execute_command(command):
+    p = subprocess.Popen(command, shell=True)
+    p.communicate()
+    if p.returncode is not 0:
+        raise Exception("Command exited with status {0}: {1}".format(
+                p.returncode, command))
+                
+
+def get_args():
+    parser = argparse.ArgumentParser(description="Generate wave samples based on wav.scp.\n"
+                                                 "Usage: gen_wav_sample.py [options...] <wav.scp> <out-data-dir>\n"
+                                                 "E.g. python gen_wav_sample.py wav.scp sample",
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)                
+                
+    parser.add_argument("--num", type=int, dest = "sample_num", default = 10, 
+                                        help="Number of samples to generate.")
+
+    parser.add_argument("--prefix", type=str, dest = "prefix", default = "", 
+                                        help="Prefix of generated samples.")
+                                        
+    parser.add_argument("--record_file", type=str, dest = "record_file", default = "", 
+                                        help="A list of records to generate samples.\n"
+                                            " The format is like this:"
+                                            "rvb_noise_f001_000008.wav \n"
+                                            "rvb_noise_f001_000009.wav \n"
+                                            "...")
+                                        
+    parser.add_argument("scp_file",
+                        help="Input data")
+    parser.add_argument("output_dir",
+                        help="Output data directory")
+                                        
```

Comment:
Indent appears inconsistent. Might help to look at some recent Kaldi python scripts for guidance.